### PR TITLE
Bugfix/claiming

### PR DESCRIPTION
**Pull request by vipergtsrz**
Fixed up a lot of the claiming component update issues after being able to test the contract claiming functionality

### CODE REVIEW

File: src/entities/DealTokenSwap.ts [hunk 1]
```diff
@@ -571,9 +572,9 @@ export class DealTokenSwap implements IDeal {
   private async hydrateDaoClaims(): Promise<void> {
     const daoTokenClaims = new Map<IDAO, Array<IDaoClaim>>();
 
-    daoTokenClaims.set(this.primaryDao, await this.getDaoClaims(this.primaryDao));
+    daoTokenClaims.set(this.primaryDao, await this.getDaoClaims(this.primaryDao, this.partnerDao));
```
Comment (dkent600) [inline src/entities/DealTokenSwap.ts:575]: would be good to comment this

File: src/entities/DealTokenSwap.ts [hunk 2]
```diff
@@ -760,6 +761,7 @@ export class DealTokenSwap implements IDeal {
           if (receipt) {
             this.isExecuted = true;
             this.executedAt = new Date((await this.ethereumService.getBlock(receipt.blockNumber)).timestamp * 1000);
+            this.hydrate();
```
Comment (dkent600) [inline src/entities/DealTokenSwap.ts:764]: What specifically needs to be hydrated here?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/entities/DealTokenSwap.ts (hunk 1 above)
Comment (vipergtsrz) [reply src/entities/DealTokenSwap.ts:575]: Added comments

File: src/entities/DealTokenSwap.ts (hunk 2 above)
Comment (vipergtsrz) [reply src/entities/DealTokenSwap.ts]: Removed hydrate(). This was used for testing to see why another component wouldn't update and I forgot to remove it. Thanks for catching that!

**Review by dkent600 — APPROVED** [review]

**Pull request merged**